pipeline_tools/input_utils.py

```diff
@@ -218,3 +219,59 @@ def create_optimus_input_tsv(uuid, version, dss_url):
         f.write('{0}'.format(sample_id))
 
     print('Finished writing files')
+
+
+def get_cellranger_inputs(uuid, version, dss_url):
```

Comment:
nit: to align with other functions, could we name this to something like: `create_cellranger_input_files`?

Reply:
By the way, are there any reasons stop us from using "TSV"? I guess we could consolidate the 2 txt files into one if we use tsv?

Reply:
I created the `txt` files to be consistent with the behavior for Optimus, which...even though it does not create a tsv file is named `create_optimus_input_tsv`

We could use a tsv file here instead, but I think this is more readable. Also, we may want to change this approach in the future so that we are downloading the fastq files in `GetInputs` instead of relying on the fragile String --> File conversion with `read_lines` so I prefer to leave this as it is for now.